editor-components/header/header8/header8.module.scss

```diff
@@ -1,214 +1,273 @@
-.container{
+@import '../../../composer-base-components/base/utitilities/viewports.module.scss';
+@keyframes slideBlur {
+  0% {
+    filter: blur(15px);
+    transform: translateX(100%);
+  }
+  100% {
+    filter: blur(0px);
+    transform: translateX(0);
+  }
+}
+.container {
   width: 100%;
-  height: 100vh;
-
-  .carousel{
+  position: relative;
+  padding: 0px;
+  .carousel, .carousel-no-image {
     width: 100%;
-    height: 100vh;
-    .slide{
+    :global(.slick-active) {
+      z-index: 1 !important;
+    }
+    .slide, .slide-no-image {
       width: 100%;
       height: 100vh;
-      filter: blur(15px);
-      transition: all 1s linear;
-      &::after{
-        background-color: rgba(0,0,0,0.3);
-        content: "";
-        width: 100%;
+      transition: filter 1s linear, transform 1s linear;
+      .image-wrapper, .image-wrapper-overlay{
+        position: relative;
         height: 100%;
-        position: absolute;
-        left: 0px;
-        top: 0px;
-      }
-      .image{
         width: 100%;
-        height: 100%;
+        &::after {
+          background-color: transparent;
+          content: "";
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+        }
+        .image {
+          width: 100%;
+          height: 100%;
+          object-fit: cover;
+        }
       }
-      &.active{
+      .image-wrapper-overlay{
+        &::after {
+          background-color: var(--composer-overlay-color);
+          content: "";
+          width: 100%;
+          height: 100%;
+          position: absolute;
+          left: 0;
+          top: 0;
+        }
+      }
+      &.active {
         filter: blur(0px);
+        transform: translateX(0);
+      }
+
+      &.active .image {
+        clip-path: inset(0);
+        animation: slideBlur 1s forwards;
+ 
+      }
+      &.active .image-ovelay {
+        clip-path: inset(0);
+        animation: slideBlur 1s forwards;
       }
     }
+    .slide-no-image{
+      height: 100%;
+    }
+    .slide-disabled-animate {
+      width: 100%;
+      height: 100vh;
+    }
   }
-
-  .max-content{
-    max-width: var(--composer-content-width);
+  .carousel-no-image{
+    position: static !important;
+  }
+  .max-content, .max-content-no-image {
     width: 100%;
     position: absolute;
-    bottom: 90px;
+    bottom: 0px;
     left: 50%;
     transform: translateX(-50%);
-    
+    padding: 20px 50px;
```

Comment:
cant you use gap value?

Reply:
I use this variable for the static container value, so I think I should not use the gap variable. We also talked about this issue.